manifold/src/manifold.cpp

```diff
@@ -264,16 +261,18 @@ Curvature Manifold::GetCurvature() const { return pImpl_->GetCurvature(); }
  * >= 0, indicating it is a new vertex. If the index is < 0, this indicates it
  * is an original vertex, the index + 3 vert of the referenced triangle.
  *
- * Every time a manifold is copied or combined to form a new manifold it gets a
- * new meshID to indicate that particular instance of the mesh. In order to look
- * up which input mesh a given instance came from, simply use the
- * MeshID2Original() static vector.
+ * In order to look up which input mesh a given instance come from, use the
+ * `originalID` field, i.e. `originalID[meshID]`. Users should not rely on the
+ * raw value of `meshID`.
```

Comment:
Is this comment out of date? It looks like you've now taken care of this so it's no longer exposed in the API, right?

Reply:
Yes, forgot to update this one.